functions/Environment/fn_campShowOSD.sqf

```diff
@@ -0,0 +1,131 @@
+/*
+ * client side function that displays message of current location with time and
+ * possibly some other related info.
+ *
+ * Parameter(s):
+ * _this select 0: array (optional)    - position (default: player's position)
+ * _this select 1: array (optional)    - date in format [_year,_month,_day,_hour,_min] (default: current date)
+ * _this select 2: string (optional)   - optional message
+ *
+ * A3 fonts used:
+ *
+ * PuristaLight
+ * PuristaMedium
+ * PuristaSemiBold
+ * PuristaBold
+ *
+ * Following example displays current player position and time:
+ *
+ * [] call A3MT_fnc_campShowOSD;
+ *
+ * Note that function requires to be run in scheduled environment with call,
+ * or use spawn in unscheduled.
+ *
+*/
+
+//get current location text
+private _get_location = {
+    params ["_position"];
```

Comment:
Don't pass 1-member array, just pass the argument directly. You can then use `_this` instead of a custom private variable - if you still want it, `private _position = _this;`